e2e-tests/playwright/e2e/authProviders/github-happy-path.spec.ts

```diff
@@ -7,124 +7,238 @@ import {
+      process.env.AUTH_PROVIDERS_GH_ORG_APP_ID,
+    );
+    deployment.addSecretData(
+      "AUTH_PROVIDERS_GH_ORG1_PRIVATE_KEY",
+      process.env.AUTH_PROVIDERS_GH_ORG1_PRIVATE_KEY,
+    );
+    deployment.addSecretData(
+      "AUTH_PROVIDERS_GH_ORG_WEBHOOK_SECRET",
+      process.env.AUTH_PROVIDERS_GH_ORG_WEBHOOK_SECRET,
+    );
 
-  test("Setup Github authentication provider and wait for first sync", async () => {
-    test.setTimeout(300 * 1000);
+    await deployment.createSecret();
 
-    LOGGER.info(`Base Url is ${process.env.BASE_URL}`);
-    LOGGER.info(
-      "Execute testcase: Setup Github authentication provider and wait for first sync",
-    );
+    // enable keycloak login with ingestion
+    await deployment.enableGithubLoginWithIngestion();
+    await deployment.setGithubResolver("usernameMatchingUserEntityName", true);
 
-    await HelmActions.upgradeHelmChartWithWait(
-      constants.AUTH_PROVIDERS_RELEASE,
-      constants.AUTH_PROVIDERS_CHART,
-      constants.AUTH_PROVIDERS_NAMESPACE,
-      constants.AUTH_PROVIDERS_VALUES_FILE,
-      constants.CHART_VERSION,
-      constants.QUAY_REPO,
-      constants.TAG_NAME,
-      [
-        "--set upstream.backstage.appConfig.signInPage=github",
-        "--set upstream.backstage.appConfig.auth.environment=production",
-        "--set upstream.backstage.appConfig.catalog.providers.githubOrg[0].orgs[0]=janus-qe",
-        "--set upstream.backstage.appConfig.catalog.providers.microsoftGraphOrg=null",
-        "--set upstream.backstage.appConfig.catalog.providers.keycloakOrg=null",
-        "--set upstream.backstage.appConfig.auth.providers.microsoft=null",
-        "--set upstream.backstage.appConfig.auth.providers.oidc=null",
-        "--set upstream.backstage.appConfig.permission.enabled=false",
-        "--set upstream.postgresql.primary.persistence.enabled=false",
-        `--set-json global.dynamic.plugins='[
-          {
-            "package": "./dynamic-plugins/dist/backstage-plugin-scaffolder-backend-module-github-dynamic",
-            "disabled": false
+    // enable required plugins and configs
+    deployment.setDynamicPluginEnabled(
+      "./dynamic-plugins/dist/backstage-plugin-scaffolder-backend-module-github-dynamic",
+      true,
+    );
+    deployment.setDynamicPluginEnabled(
+      "./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-dynamic",
+      true,
+    );
+    deployment.setDynamicPluginEnabled(
+      "./dynamic-plugins/dist/backstage-community-plugin-github-issues",
+      true,
+    );
+    deployment.setDynamicPluginEnabled(
+      "./dynamic-plugins/dist/roadiehq-backstage-plugin-github-pull-requests",
+      true,
+    );
+    deployment.setDynamicPluginEnabled(
+      "./dynamic-plugins/dist/backstage-community-plugin-github-actions",
+      true,
+    );
+    deployment.setDynamicPluginEnabled(
+      "./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-org-dynamic",
+      true,
+    );
```

Comment:
```suggestion
    const dynamicPlugins =  [
      "./dynamic-plugins/dist/backstage-plugin-scaffolder-backend-module-github-dynamic",
      "./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-dynamic",
      "./dynamic-plugins/dist/backstage-community-plugin-github-issues",
      "./dynamic-plugins/dist/roadiehq-backstage-plugin-github-pull-requests",
      "./dynamic-plugins/dist/backstage-community-plugin-github-actions",
      "./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-org-dynamic",
    ]

    dynamicPlugins.forEach((pluginName) =>  {
      deployment.setDynamicPluginEnabled(pluginName, true)
    });
```

Would it be better to loop through enabling a list of dynamic plugins?